stories/topics/con-aws-restore-process.adoc

```diff
@@ -4,6 +4,11 @@
 
 Use the following procedures to ensure a smooth restore process.
 
+[NOTE]
+====
+A restored deployment will contain the same VPC networking setup as the original backed up deployment. If a backed up deployment was deployed within an existing VPC, its restored deployment will also be deployed into that VPC. If a backed up deployment was deployed with a new VPC, its restored deployment will also be deployed with a new VPC.  
```

Comment:
A restored deployment contains the same VPC....

its resorted deployment is also deployed into... 

and again in the next sentence